### PR TITLE
symantec_endpoint: use ip_address field to populate related.ip

### LINKED ISSUE #6500: [symantec_endpoint] symantec_endpoint.log.ip_address
For logs relating to completed AV scans the IP of the client is ingested into symantec_endpoint.log.ip_address

Two things:

* This field seems to be typed as keyword, should it not be IP?
* This is not added to related.ip - seems like this should probably be added?

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Uses the `symantec_endpoint.log.ip_address` and `symantec_endpoint.log.remote_host_ip` fields were possible. Also adds a test for the case where `remote_host_ip` is not contaminated with extra characters (waiting on provenance of the test case to determine whether the case should be cleaned or more effort should be put into extracting the address).

Does not convert the original fields to `ip` in case they are mal-formed, so users can examine them in those cases.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes #6500

### CODE REVIEW

File: packages/symantec_endpoint/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -1099,6 +1110,7 @@ processors:
       - _conf
       - _csv_array
       - _fingerprint
+      - _temp
```
Comment (andrewkroh) [inline packages/symantec_endpoint/data_stream/log/elasticsearch/ingest_pipeline/default.yml:1113]: There is also a `remove` in the on_failure that performs the same cleanup. Can you update that too. I wish there was a `finally` block.

File: packages/symantec_endpoint/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.7.0"
+  changes:
+    - description: Use `ip_address` and `remote_host_ip` fields.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```
Comment (andrewkroh) [inline packages/symantec_endpoint/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/6549
```

File: packages/symantec_endpoint/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.7.0"
+  changes:
+    - description: Use `ip_address` and `remote_host_ip` fields.
```
Comment (andrewkroh) [inline packages/symantec_endpoint/changelog.yml:4]: I feel like this is missing some detail like "Use ip_address to populate related.ip."

**Review by andrewkroh — COMMENTED** [review]
This mentions the `remote_host_ip` field, but I don't see any change that would affect that field or related fields. Did I mess it?

**Comment by efd6** [comment]
You did not miss anything. Originally I thought remote_host_ip was not being used, but this was because of the dot after the IP in the test case that I added to.

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**